nshutijonathan/Auto-Mart
# #166175515 set up the node server

#### What does this PR do?
Have a read on resource endpoint in Auto-Mart REST API
#### Description of Task to be completed?
Have This endpoint working
* GET/localhost:3000

Have ES6 Implementation with Babel

Have Linting library configured(Eslint)

Have Testing framework configured(Mocha)
#### How should this be manually tested?
* Download or clone the tutorial project code from [this repo](https://github.com/nshutijonathan/Auto-Mart), Install all required NPM  packages by running NPM install from the command line in the project root folder (where the package.json is located).
Start the API by running NPM  start from the command line in the project root folder, you should see the message Server listening on port 3000. You can test the API directly using an application such as Postman or you can test it with one of the single pages in the browser.
#### Any background context you want to provide?
* There is Implementation of Javascript(ES6) logic with data structures to persist some data
#### What are the relevant pivotal tracker stories?
[#166175515](https://www.pivotaltracker.com/n/projects/2346053/stories/166175515)
--
